eng/common/TestResources/sub-config/AzurePublicMsft.json

```diff
@@ -2,8 +2,10 @@
     "SubscriptionId": "2cd617ea-1866-46b1-90e3-fffb087ebf9b",
     "TenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
     "TestApplicationId": "f850650c-1fcf-4489-b46f-71af2e30d360",
+    "TestApplicationSecret": "",
```

Comment:
Why did these get added? Is that because you want to them to nullify the secret if it exists?

Reply:
Yes. This cleans them out of a `$(sub-config...)` specified earlier in the chain.